config/environments/development.rb

```diff
@@ -3,6 +3,14 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Allow additional hosts to connect. Provide hostnames as a comma-separated list.
+  ENV.fetch('PERMITTED_HOSTS', nil).tap do |permitted_hosts|
```

Comment:
Rails 7 introduced the `RAILS_DEVELOPMENT_HOSTS` environment variable for doing this in https://github.com/rails/rails/pull/41560. Can we use the same environment variable name and add a comment that we should remove this when upgrading to Rails 7?

Reply:
Yes.